stdlib/public/stubs/Stubs.cpp

```diff
@@ -327,6 +340,52 @@ extern "C" long double _swift_fmodl(long double lhs, long double rhs) {
     return fmodl(lhs, rhs);
 }
 
+#if defined(__APPLE__)
+SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_INTERFACE
+extern "C" char ** _swift_stdlib_getUnsafeArgcArgv(int *outArgLen) {
+  assert(outArgLen != nullptr);
+
+  *outArgLen = *_NSGetArgc();
+  return *_NSGetArgv();
+}
+#elif defined(__linux__) || defined(__CYGWIN__) || defined(__FreeBSD__)
+SWIFT_CC(swift) SWIFT_RUNTIME_STDLIB_INTERFACE
+extern "C" char ** _swift_stdlib_getUnsafeArgcArgv(int *outArgLen) {
+  assert(outArgLen != nullptr);
+  
+  FILE *cmdline = fopen("/proc/self/cmdline", "rb");
+  if (!cmdline) {
+    swift::fatalError(0,
+                      "fatal error: Unable to open command line interface.\n");
```

Comment:
I think using `/proc/self/cmdline` in the error will help a lot of people who run Swift in restricted environments.  Seeing a message about `/proc` makes it immediately obvious what's wrong (`/proc` not available, or not mounted etc.)